dangerzone/conversion/doc_to_pixels_qubes_wrapper.py

```diff
@@ -14,36 +15,24 @@
     from doc_to_pixels import DocumentToPixels
 
 
-def recv_b():
-    """Qrexec wrapper for receiving binary data from the client
-
-    Borrowed from https://github.com/QubesOS/qubes-app-linux-pdf-converter/blob/main/qubespdfconverter/server.py#L82
-    """
-    untrusted_data = sys.stdin.buffer.read()
-    if not untrusted_data:
+def read_bytes() -> bytes:
+    """Read bytes from the stdin."""
+    data = sys.stdin.buffer.read()
+    if data is None:
         raise EOFError
-    return untrusted_data
-
-
-def send_b(data):
-    """Qrexec wrapper for sending binary data to the client
+    return data
 
-    Borrowed from https://github.com/QubesOS/qubes-app-linux-pdf-converter/blob/main/qubespdfconverter/server.py#L82
 
-    """
-    if isinstance(data, (str, int)):
-        data = str(data).encode()
+def write_bytes(data: bytes, file=sys.stdout):
+    file.buffer.write(data)
 
-    sys.stdout.buffer.write(data)
-    sys.stdout.buffer.flush()
 
+def write_text(text: str, file=sys.stdout):
+    write_bytes(text.encode(), file=file)
 
-def send(data):
-    """Qrexec wrapper for sending text data to the client
 
-    borrowed from https://github.com/QubesOS/qubes-app-linux-pdf-converter/blob/main/qubespdfconverter/server.py#L77
-    """
-    print(data, flush=True)
+def write_int(num: int, file=sys.stdout):
+    write_bytes(num.to_bytes(2, signed=False), file=file)
```

Comment:
Explicit endianness :

```suggestion
    write_bytes(num.to_bytes(2, byteorder="big", signed=False), file=file)
```

Reply:
I didn't specify this because big endian is pretty common nowadays, and the default in Python.

Reply:
Solved in https://github.com/freedomofpress/dangerzone/pull/437/commits/5bb695f91c57805375b6473bf6097a9435b86436